meta_aggregation_api/services/limit_orders.py

```diff
@@ -91,12 +101,12 @@ async def post(
         provider_class = all_providers.get(provider)
         if not provider_class:
             raise ProviderNotFound(provider)
-        provider_instance = provider_class()
+        provider_instance = provider_class(self.session, self.config, self.apm_client)
         logger.info(
             f'Posting limit order: {order_hash}',
             extra={'provider': provider.__class__.__name__, 'order_hash': order_hash},
         )
-        response = await provider_instance.post_limit_order(
+        response = await provider_instance.post_limit_order(  # TODO: add the method to the base class?
```

Comment:
@shestakoven Unresolved attribute reference 'get_order_by_hash' for class 'BaseProvider'

Reply:
^

Reply:
fixed in the latest commits